server/app/services/cloud/azure/AzurePublicStorage.java

```diff
@@ -8,14 +8,16 @@ public class AzurePublicStorage extends PublicStorageClient {
   @Override
   public StorageUploadRequest getSignedUploadRequest(
       String fileKey, String successRedirectActionLink) {
-    // TODO(#5676): Implement for Azure.
```

Comment:
I've removed these TODOs about implementing for Azure because Bion and I decided to not spend time on any Azure implementation since no current or future deployments are planning to use Azure.